private/bufpkg/bufplugin/bufplugin.go

```diff
@@ -311,7 +311,7 @@ func MergeInsertionPoints(responses []*pluginpb.CodeGeneratorResponse) ([]Merged
 				continue
 			}
 			if _, ok := allFiles[fileName]; ok {
-				return nil, fmt.Errorf("file %q defined multiple times", fileName)
+				return nil, fmt.Errorf("file %q was generated multiple times", fileName)
```

Comment:
I think it'd be even better if we could get the names of the plugins that caused the responses. What do you think about changing the parameter to this function to take a wrapper that contains the plugin name as well as the response?

Reply:
Yea I was thinking about this and was originally dissuaded since the `PluginConfig` types differed between this usage and the one in `bufbuild/core`. But given that we already have a `bufplugin` type for `MergedPluginResult`, it seems fine to include a symmetrical request type.

I'll check this out in a follow-up and pair it along with the open issue related to this error in particular.